domains/draunts.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "CxmboFight"
+    },
+    "records": {
+        "CNAME": "cxmbofight.github.io"
```

Comment:
Site does not return any preview or data. No preview was provided in the pull request.